src/Microsoft.Azure.WebJobs.Host/Triggers/TriggerArgumentBinding/StringTriggerArgumentBinding.cs

```diff
@@ -3,13 +3,13 @@
 
 using System.Collections.Generic;
 
-namespace Microsoft.Azure.WebJobs.ServiceBus
+namespace Microsoft.Azure.WebJobs.Host.Triggers
 {
     // Bind TMessage --> String. USe IConverterManager for the conversion. 
     internal class StringTriggerArgumentBinding<TMessage, TTriggerValue> : SimpleTriggerArgumentBinding<TMessage, TTriggerValue>
```

Comment:
It's strange that the type 'string' doesn't show up in the generics anywhere - if this is a string binding, why isn't it strongly typed?